sources/packages/api/src/route-controllers/program-info-request/program-info-request.controller.ts

```diff
@@ -281,14 +281,17 @@ export class ProgramInfoRequestController extends BaseController {
         } as EducationProgramOffering;
       }
 
+      const studyStartDate = new Date(payload.studentStudyStartDate);
```

Comment:
I believe that if the offering is present (line 266) the studentStudyStartDate and studentStudyEndDate must come from the offering and not the payload.

Reply:
I can see from the code further down that the offering might still need the studyStartDate and studyEndDate to come from the payload which will still be undefined.

<img width="610" alt="image" src="https://user-images.githubusercontent.com/72251620/163442066-96e0d43e-dba8-4947-9c4e-6c0afe36c610.png">